rct229/utils/json_utils.py

```diff
@@ -1,11 +1,36 @@
 import json
-
+import logging
+import sys
 
 def load_json(json_file_path):
     with open(json_file_path) as f:
         return json.load(f)
 
 
+def get_value_from_key(key, context, func_name=None):
```

Comment:
Added this function to handle the key reading from a dict.

The get function returns None if key is not exist.

Use response to return the error message to parent routine call.